BinaryTree.java

```diff
@@ -0,0 +1,68 @@
+package consistent;
+
+/**
+ * Created by deepakrtp on 01/04/17.
+ */
+import io.netty.bootstrap.Bootstrap;
+
+
+public class BinaryTree {
+
+    public final int NOOFSERVERS = 8;
```

Comment:
Why did you choose this way of doing it?
Better way of doing it would be to have an array of 7 nodes and name the servers with 0,1,2,3,4,5,6 (instead of 1,2,3,4,5,6,7). Or if you are particular about naming the servers 1,2,3,4,5,6,7 you can subtract the server id by 1 to get the index of the node. This way of keeping 8 nodes and leaving node 0 empty seems like a hack.